### PR TITLE
UX: Rename private to personal messages in the interface

### PR DESCRIPTION
**Pull request by moin-Jana**
This PR ensures consistent terminology across the interface by replacing "private message" with "personal message" in user-facing elements.

### CODE REVIEW

File: config/locales/client.en.yml [hunk 1]
```diff
@@ -3554,7 +3554,7 @@ en:
 
       flag_topic:
         title: "Flag"
-        help: "privately flag this topic for attention or send a private notification about it"
```
Comment (tobiaseigen) [inline config/locales/client.en.yml:3557]: I think we do want to make clear that flags are not public. How would you change this to make that clear without using the word "privately" here? I agree with changing "private notification" to "personal message".

File: config/locales/client.en.yml [hunk 2]
```diff
@@ -3862,7 +3862,7 @@ en:
         edit_action: "Edit"
         edit_anonymous: "Sorry, but you need to be logged in to edit this post."
         flag_action: "Flag"
-        flag: "privately flag this post for attention or send a private notification about it"
```
Comment (tobiaseigen) [inline config/locales/client.en.yml:3865]: Same thing here - I think we want to keep "privately" or find way to make clear that flags are not public, and are only seen by moderators and admins.

File: config/locales/client.en.yml [hunk 3]
```diff
@@ -4212,10 +4212,10 @@ en:
       yes_delete_spammer: "Yes, Delete Spammer"
       ip_address_missing: "(N/A)"
       hidden_email_address: "(hidden)"
-      submit_tooltip: "Submit the private flag"
+      submit_tooltip: "Submit the flag"
       take_action_tooltip: "Reach the flag threshold immediately, hide the post, and agree with all pending flags"
       cant: "Sorry, you can't flag this post at this time."
-      notify_staff: "Notify staff privately"
```
Comment (tobiaseigen) [inline config/locales/client.en.yml:4218]: Again, I think we want to keep "privately" here unless you have a suggestion on a better formulation. Without "privately" it is not totally clear what happens with the flag and the user might think the flag will be public.

File: plugins/discourse-narrative-bot/config/locales/server.en.yml [hunk 4]
```diff
@@ -302,13 +302,13 @@ en:
...
+          We like our discussions friendly, and we need your help to [keep things civilized](%{guidelines_url}). If you see a problem, please flag to let the author or [our helpful staff](%{about_url}) know about it. There are many reasons you might want to flag a post, ranging from an innocuous topic-splitting suggestion to a clear-cut community standards violation. If you select **Something Else**, you'll start a personal message discussion with the moderators where you can ask further questions.
 
           > :imp: I wrote something nasty here
 
           Go ahead and **flag this post** <img src="%{base_uri}/plugins/discourse-narrative-bot/images/font-awesome-flag.png" width="16" height="16"> and select **It's Inappropriate** as the reason!
         reply: |-
-          [Our staff](%{base_uri}/groups/staff) will be privately notified about your flag. If enough community members flag a post, it will also be automatically hidden as a precaution. (Since I didn’t actually write a nasty post :angel:, I’ve gone ahead and removed the flag for now.)
```
Comment (tobiaseigen) [inline plugins/discourse-narrative-bot/config/locales/server.en.yml:311]: oops I found one more - can you add this "privately" back?

**Review by hlashbrooke — APPROVED** [review]

**Pull request merged**